card/lib/card/loader.rb

```diff
@@ -21,6 +21,40 @@ def load_mods
         load_set_patterns
         load_formats
         load_sets
+
+        if ENV['RAILS_ENV'] == "development"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.